### ISSUE TITLE
Compilation problem on OpenSUSE LEAP 42.1(x86_64)

### ISSUE THREAD

**Issue by Athas** [bh107/bohrium#71]
It seems that Bohrium assumes that it can find libraries in `lib` directories, but on OpenSUSE x86_64, the proper libraries are in `lib64` directories.  This leads to interesting situations such as finding a 32-bit `libGL.so` in `/usr/X11R6/lib`, and then failing during `make` because the library has the wrong architecture.

If I fix this by manually symlinking `/usr/X11R6/lib/libGL.so` to a 64-bit version (don't judge), Bohrium will compile, but fail with an error during `make install`:

```
CMake Error at bridge/npbackend/cmake_install.cmake:36 (file):
  file INSTALL cannot find "/home/athas/bohrium2/build/bridge/npbackend/lib".
Call Stack (most recent call first):
  cmake_install.cmake:71 (include)
```

It is true that `build/bridge/npbackend/lib` does not exist, but `build/bridge/npbackend/lib64` does, so I think these two problems have the same underlying source.

While I fully understand that supporting SCIENCE minus ITs former favourite operating system OpenSUSE may not be high on your priority list, I think SCIENCE minus ITs new favourite operating system Red Hat uses the same directory structure, and so might have the same problem.

If you need an OpenSUSE system to test on, let me know and I'll be happy to provide.

**Comment by madsbk** [comment]
Did b419aac fix the problem?

**Comment by Athas** [comment]
Seems to compile out-of-the-box now.

**Comment by madsbk** [comment]
Fixed in b419aac
